paddle/fluid/operators/elementwise/elementwise_mul_op.h

```diff
@@ -22,56 +23,15 @@ limitations under the License. */
 namespace paddle {
 namespace operators {
 
-class ElementwiseMulOp : public framework::OperatorWithKernel {
+class ElementwiseMulOp : public ElementwiseOp {
  public:
-  using framework::OperatorWithKernel::OperatorWithKernel;
```

Comment:
You can just try 
```using ElementwiseOp::ElementwiseOp;
```
and no need to extend ElementwiseOp construct func.

Reply:
Yup, that makes it compile just fine. I didn't know that one, thank you so much for your suggestion, will appear in the next commit.